centreon/tests/e2e/features/Cloud-notifications/01-notification-create.feature

```diff
@@ -23,6 +23,25 @@ Feature: Creating a Notification Rule
     And the notification refresh_delay has been reached
     Then an email is sent to the configured '<contact_settings>' with the configured format
     Examples:
-      | contact_settings                            | resource_type                           |
-      | a single contact                            | host group                              |
-      | two contacts                                | host group and services for these hosts |
+      | contact_settings | resource_type                           |
+      | a single contact | host group                              |
+      | two contacts     | host group and services for these hosts |
+
+  @ignore
+  Scenario Outline: Creating a large volume Notification Rule for <contact_settings>
+    Given a minimum of 1000 services linked to the selected <resource_type> and <contact_settings>
+    When the user defines a name for the rule
+    And the user selects a <resource_type> with associated events on which to notify
+    And the user selects the <contact_settings>
+    And the user defines a mail subject
+    And the user defines a mail body
+    And the user clicks on the "Save" button to confirm
+    Then a success message is displayed and the created Notification Rule is displayed in the listing
+    When changes occur in the configured statuses for the selected <resource_type>
+    And the hard state has been reached
+    And the notification refresh_delay has been reached
+    Then an email is sent to the configured <contact_settings> with the configured format
+    Examples:
+      | contact_settings | resource_type                           |
+      | a single contact | host group                              |
+      | two contacts     | host group and services for these hosts |
```

Comment:
`a minimum of 1000 services linked to the selected host group and services for these hosts and two contacts` is not understandable